index.md

```diff
@@ -26,4 +29,9 @@ Welcome to this list of Malagasy artists. If you want to know the famous artists
 - [W](w/index.md)
 - [X](x/index.md)
 - [Y](y/index.md)
-- [Z](z/index.md)
\ No newline at end of file
+- [Z](z/index.md)
+
+## Genres
+There are some genres wich are originated from each region in Madagascar, to know more click the link below.
```

Comment:
Small typo here too, it should read: `which originated`

Reply:
We've sent the last change.